charts/nri-metadata-injection/values.yaml

```diff
@@ -3,15 +3,6 @@ nameOverride: ""
 # -- Override the full name of the release
 fullnameOverride: ""
 
-# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`
-cluster: ""
```

Comment:
this is needed I believe because that info is injected as env var

Reply:
Solved in [#e4fba9f](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/e4fba9f678497857c8c3ba587ae9be175ed5398d)